backend/app/infrastructure/routes/user.py

```diff
@@ -152,6 +170,47 @@ def patch(self, user_address: str):
         return {"status": patron_mode_status}, 200
 
 
+@ns.route("/<string:user_address>/antisybil-status")
+@ns.doc(
+    params={
+        "user_address": "User ethereum address in hexadecimal format (case-insensitive, prefixed with 0x)"
+    }
+)
+class AntisybilStatus(OctantResource):
+    @ns.doc(
+        description="Returns user's antisybil status.",
+    )
+    @ns.marshal_with(user_antisybil_status_model)
+    @ns.response(200, "User's cached antisybil status retrieved")
+    def get(self, user_address: str):
+        app.logger.debug(f"Getting user {user_address} cached antisybil status")
+        antisybil_status = get_user_antisybil_status(user_address)
+        app.logger.debug(f"User {user_address} antisybil status: {antisybil_status}")
+        if antisybil_status is None:
```

Comment:
we should throw antysibil_status for being not found as 404 Exception as weel, as we did with users; but you can put it in  TODO section, you don't have to change this in the PR since we're in a hurry with delivering it 👌

Reply:
I'll resolve this one for now.